### ISSUE TITLE
Different Behaviors for demo.py

### ISSUE THREAD

**Issue by Jasonlee1995** [PeterWang512/CNNDetection#17]
Hi. 

I'm interested on generalization quality of the model so do some experiments and found different behaviors on doing tests.

For computing expenses, I just download your test dataset - StyleGAN2 - LSUN cat class and run the experiments.

I get good score as similar as your paper and repo as below figure.

![image](https://user-images.githubusercontent.com/49643709/124215527-2378d780-db2f-11eb-8e2c-1e6d18a6e706.png)
![image](https://user-images.githubusercontent.com/49643709/124215543-2a9fe580-db2f-11eb-9aad-a1f7ee7cf4ca.png)

But found different behavior when I do demo on generated stylegan2 images from official repo... (not from the downloaded images you provide)

![image](https://user-images.githubusercontent.com/49643709/124215474-0a702680-db2f-11eb-9ff5-79872d9764e5.png)
![image](https://user-images.githubusercontent.com/49643709/124215497-165be880-db2f-11eb-8d6d-52dba8dac8c8.png)

I don't know why the model behave differently... (do the same codes as downloaded images)

I generate images using official repo [[link]](https://github.com/NVlabs/stylegan2-ada-pytorch) legacy.py
![image](https://user-images.githubusercontent.com/49643709/124215737-971ae480-db2f-11eb-9cac-da8ffb3bc356.png)

I upload the google drive open links for more details for downloads and also the testing codes (ipynb files)
[google drive link](https://drive.google.com/drive/u/1/folders/1RP30wEPKjWhkxrGhDtgGSBi7xWYFbnrM)

Too much performance drops on same model (StyleGAN2) with dataset (LSUN dataset with only cat class) and I think I'm missing some core parts of detecting real/fake and I don't know what I'm missing... Can you advice about more details or core parts that I'm missing?

**Comment by PeterWang512** [comment]
The difference might come from how you are saving the images and the truncation used. We collected stylegan2 images by directly downloading from images released officially: https://drive.google.com/drive/folders/1BA2OZ1GshdfFZGYZPob5QWOGBuJCdu5q

**Comment by Jasonlee1995** [comment]
> The difference might come from how you are saving the images and the truncation used. We collected stylegan2 images by directly downloading from images released officially: https://drive.google.com/drive/folders/1BA2OZ1GshdfFZGYZPob5QWOGBuJCdu5q
> We chose samples with truncation 0.5 for more realistic-looking fakes.

I additionally download the google drive from the folders both psi 0.5 and 1, and noticed that it just works fine.

I think performance degradation might be caused by pre-trained weights of StyleGAN2 or saving process maybe?? (although it is from official repo)

I'll add more details and if I found something

![psi 0.5 with 0.1 weight](https://user-images.githubusercontent.com/49643709/124268351-5e512e80-db74-11eb-98a7-6d8241ecc3e8.png)
![psi 0.5 with 0.5 weight](https://user-images.githubusercontent.com/49643709/124268401-6ad58700-db74-11eb-98c4-b24ea32b734f.png)
![psi 1 with 0.1 weight](https://user-images.githubusercontent.com/49643709/124268453-7c1e9380-db74-11eb-88c1-61d90a06d6f5.png)
![psi 1 with 0.5 weight](https://user-images.githubusercontent.com/49643709/124268484-86d92880-db74-11eb-94f4-96cbd6f5b35b.png)

**Comment by Jasonlee1995** [comment]
The issues appear cause of without normalization and save.
StyleGAN2 model outputs are in range(-1, 1) but just save images of these tensors cause flaws.
Change the range(-1, 1) to range(0, 1) and save just works fine :)
